src/Numerics/GridBConds.h

```diff
@@ -84,15 +86,8 @@ struct GridBCond
   {
     if(periodic)
     {
-#if defined(HAVE_STD_ROUND)
       T x1=x*OneOverL;
       x=L*(x1-round(x1));
```

Comment:
This may not call the right 'round' function if type T is float (In mixed precision mode, does this code get used as float?)
For C++11, this could be std::round
If C++11 is not enabled, it should use roundf in single precision.